AudioSinkExample/AudioSinks/ExampleAudioRecorder.h

```diff
@@ -0,0 +1,22 @@
+//
+//  ExampleAudioRecorder.h
+//  AudioSinkExample
+//
+//  Copyright © 2017 Twilio, Inc. All rights reserved.
+//
+
+@import Foundation;
+
+#import <TwilioVideo/TwilioVideo.h>
```

Comment:
Use `@import TwilioVideo` instead of importing the header. The  `CallKit` category implementation warning should go away.

Reply:
Worked, thanks!